test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-plus-quantifier-flags-g.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018 Leo Balter.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: prod-CharacterClassEscape
+description: >
+    Compare range for Digit class escape, \\d+ with flags g
+info: |
+    This is a generated test, please checkout https://github.com/bocoup/test262-regexp-generator
+    for any changes.
+
+    CharacterClassEscape[U] ::
+        d
+        D
+        s
+        S
+        w
+        W
+
+    21.2.2.12 CharacterClassEscape
+    
+    The production CharacterClassEscape :: d evaluates as follows:
+        Return the ten-element set of characters containing the characters 0 through 9 inclusive.
+    The production CharacterClassEscape :: D evaluates as follows:
+        Return the set of all characters not included in the set returned by CharacterClassEscape :: d.
+    The production CharacterClassEscape :: s evaluates as follows:
+        Return the set of characters containing the characters that are on the right-hand side of
+        the WhiteSpace or LineTerminator productions.
+    The production CharacterClassEscape :: S evaluates as follows:
+        Return the set of all characters not included in the set returned by CharacterClassEscape :: s.
+    The production CharacterClassEscape :: w evaluates as follows:
+        Return the set of all characters returned by WordCharacters().
+    The production CharacterClassEscape :: W evaluates as follows:
+        Return the set of all characters not included in the set returned by CharacterClassEscape :: w.
+---*/
+
+var re = /\d+/g;
+var matchingRange = /[0-9]+/g;
+var msg = '"\\u{REPLACE}" should be in range for \\d+ with flags g';
+
+var i;
+var fromEscape, fromRange, str;
+for (i = 0; i < 0xFFFF; i++) {
+
+    str = String.fromCharCode(i);
```

Comment:
Nit: There’s no reason to not always just use `String.fromCodePoint`, _especially_ when you’re not going beyond U+FFFF.